### PR TITLE
Shutdown procedure

### PR DESCRIPTION
**Pull request by cTurtle98**
procedure to follow when the pool test is over

### CODE REVIEW

**Comment by cTurtle98** [comment]
@OrionOth I renamed the cables for you

File: doc/shutdown-procedure.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+1. in the third terminal window type the following to shutdown the DECK
+
+```console
+sudo shutdown now
+```
+
+1. wait for the fans to stop spinning on the deck box pc
+
+1. switch the `POWER IN` switch too `OFF`
```
Comment (OrionOth) [inline doc/shutdown-procedure.md]: suffer, dyslexic mortal

**Review by OrionOth — APPROVED** [review]
seems pretty good tbh

File: doc/shutdown-procedure.md (hunk 1 above)
Comment (cTurtle98) [reply doc/shutdown-procedure.md]: fixed it

**Pull request merged**